### ISSUE TITLE
Implement Lazyloading on the recent files list

### ISSUE THREAD

**Issue by Likqez** [files-community/Files#13186]
### What feature or improvement do you think would benefit Files?

When accessing the Home Page, the app eagerly handles the recent files list, causing the following things to happen when the list contains a file on a network share:

- quick-access widget waits for file list to load
- there are drives missing
- drive icons are not shown
- drive capacity is not shown
- app is unresponsive when starting up files (when home is default page)

Implementing lazy loading of the files list would cut time waiting on application startup, which is critical.
It would also make the app more responsive if navigating via the Home Page.

Yes this does not fix the underlying issue of the handling of network shares, but it would improve the overall situation.

### Requirements

This proposal required changing the retrieval of the recent files widget on the home page

### Files Version

2.5.21.0

### Windows Version

10.0.22621.2134

**Comment by Josh65-2201** [comment]
Thanks for the feedback, I've added this to the project board.

**Comment by yaira2** [comment]
@Likqez thank you for your feedback, recent files don't take very long to load. Regarding the issue with Network drives, we're tracking that in another issue and it's on the ToDo list to fix.
